src/Controls/tests/UITests/Tests/Issues/Issue19509.cs

```diff
@@ -13,13 +13,15 @@ public Issue19509(TestDevice device) : base(device)
 		public override string Issue => "Entry TextColor property not working when the Text value is bound after some time";
 
 		[Test]
-		public void Issue19509Test()
+		public async Task Issue19509Test()
 		{
 			App.WaitForElement("WaitForStubControl");
 
 			// 1. Click a button to update the text
 			App.Click("button");
 
+			await Task.Delay(1000); // Wait to Ripple Animation to complete
```

Comment:
Do we know how long the ripple animation generally takes to run? We are using Material button correct? We could probably look at the android source code for this to get an idea of the actual time and adjust our delay accordingly?

Reply:
That will be a bit iffy still. Sometimes the emulator chokes for a few ms and things like that. The delay is probably far less - 300-500, but that assumes the time it takes for the call to return from appium the emulator was fast and complete the ripple.

Reply:
Yep, just thinking it would be good to find it, document the expected time, and then still add some buffer....

Reply:
I found these: https://github.com/aosp-mirror/platform_frameworks_base/blob/52bd4f135acdb36fbc7f3a4402bb29416e3ba06c/graphics/java/android/graphics/drawable/RippleForeground.java#L46-L53

Reply:
Sweet let's add that to a comment above the delay. If we ever see failures here again we know what the expected delay is and can bump it up a bit more.

Reply:
Maybe just this for now?
https://github.com/dotnet/maui/pull/21823